tests/numberparsingcheck.cpp

```diff
@@ -10,6 +9,11 @@
 #define JSON_TEST_NUMBERS
 #endif
 
+#ifndef _MSC_VER
```

Comment:
Future note: we should just bite the bullet and rename dirent_portable.h to dirent.h.